apps/web/vibes/soul/sections/footer/index.tsx

```diff
@@ -61,96 +63,307 @@ export const Footer = forwardRef(function Footer(
       <div className="mx-auto max-w-screen-2xl px-4 py-6 @xl:px-6 @xl:py-10 @4xl:px-8 @4xl:py-12">
         <div className="flex flex-col justify-between gap-x-8 gap-y-12 @3xl:flex-row">
           <div className="@3xl:w-1/3">
-            {/* Contact Information */}
-            {contactInformation?.address != null || contactInformation?.phone != null ? (
-              <div className="text-xl font-medium @lg:text-2xl">
-                <h3 className="text-contrast-300">Contact Us</h3>
-                <div>
-                  {contactInformation.address != null && contactInformation.address !== '' && (
-                    <p>{contactInformation.address}</p>
-                  )}
-                  {contactInformation.phone != null && contactInformation.phone !== '' && (
-                    <p>{contactInformation.phone}</p>
-                  )}
+            {/* Logo Information */}
+            <Stream
+              fallback={
+                <div className="mb-2 flex h-10 animate-pulse items-center text-2xl">
+                  <span className="h-[1ex] w-[10ch] rounded bg-contrast-100" />
                 </div>
-              </div>
-            ) : (
-              // Logo
-              <Link
-                href="#"
-                className="relative inline-block h-10 w-full max-w-56 rounded-lg ring-primary focus-visible:outline-0 focus-visible:ring-2"
-              >
-                {typeof logo === 'string' ? (
-                  <span className="whitespace-nowrap font-heading text-2xl font-semibold">
-                    {logo}
-                  </span>
-                ) : (
-                  logo?.src != null &&
-                  logo.src !== '' && (
-                    <Image
-                      src={logo.src}
-                      fill
-                      sizes="400px"
-                      alt={logo.alt}
-                      className="object-contain object-left"
-                    />
+              }
+              value={streamableLogo}
+            >
+              {logo => {
+                if (logo != null && typeof logo === 'string') {
+                  return (
+                    <Link
+                      className="relative mb-2 inline-block h-10 w-full max-w-56 rounded-lg ring-primary focus-visible:outline-0 focus-visible:ring-2"
+                      href="#"
+                    >
+                      <span className="whitespace-nowrap font-heading text-2xl font-semibold">
+                        {logo}
+                      </span>
+                    </Link>
                   )
-                )}
-              </Link>
-            )}
-
-            {/* Social Media Links */}
-            {socialMediaLinks != null && (
-              <div className="flex items-center gap-3 pt-8 @3xl:pt-10">
-                {socialMediaLinks.map(({ href, icon }, i) => {
+                }
+                if (logo != null && typeof logo === 'object' && logo.src !== '') {
```

Comment:
Is there a reason the `logo.src` would come back as empty string? I would think logo would come back as `null` if there was no `src` property?

Reply:
I only changed a little from the original logic, but I did notice that some conditional rendering and type guards needed to be clearer/consistent. It might be worth it later to go through the project and check these patterns.